.github/workflows/t3000-perplexity-tests.yaml

```diff
@@ -0,0 +1,59 @@
+name: "(T3K) T3000 perplexity tests"
+
+on:
+  push:
```

Comment:
push on main?

Reply:
for testing the workflow, will remove when https://github.com/tenstorrent/tt-metal/actions/runs/9899640375 passes

Reply:
@tt-rkim 
passed and removed